rattr/analyser/cls.py

```diff
@@ -63,6 +63,12 @@ def is_enum(cls: ast.ClassDef) -> bool:
     return any(n == "Enum" or n.endswith(".Enum") for n in get_base_names(cls))
 
 
+def is_namedtuple(cls: ast.ClassDef) -> bool:
+    # NOTE Purely heuristic, though it does allow for user defined Enum bases
```

Comment:
```suggestion
    # NOTE Purely heuristic, though it does allow for user defined NamedTuple bases
```

Reply:
Note: the way that ruff does these heuristics is by tracking imports through the file. But I think that might be too much work for us. So ruff, in theory knows that this reference is `typing.NamedTuple` (although I think if you're sneaky and _redefine_ `NamedTuple` in the same file etc etc it won't pick it up)